x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -119,27 +102,40 @@ export const securitySolutionsDeepLinks: SecuritySolutionDeepLink[] = [
   {
     id: SecurityPageName.dashboardsLanding,
     title: DASHBOARDS,
-    path: DASHBOARDS_PATH,
-    navLinkStatus: AppNavLinkStatus.hidden,
+    path: OVERVIEW_PATH,
+    navLinkStatus: AppNavLinkStatus.visible,
+    searchable: false,
+    order: 9000,
     features: [FEATURE.general],
     keywords: [
       i18n.translate('xpack.securitySolution.search.dashboards', {
         defaultMessage: 'Dashboards',
       }),
     ],
-  },
-  {
-    id: SecurityPageName.detectionAndResponse,
-    title: DETECTION_RESPONSE,
-    path: DETECTION_RESPONSE_PATH,
-    navLinkStatus: AppNavLinkStatus.hidden,
-    features: [FEATURE.general],
-    keywords: [
-      i18n.translate('xpack.securitySolution.search.detectionAndResponse', {
-        defaultMessage: 'Detection & Response',
-      }),
+    deepLinks: [
+      {
+        id: SecurityPageName.overview,
+        title: OVERVIEW,
+        path: OVERVIEW_PATH,
+        features: [FEATURE.general],
+        keywords: [
+          i18n.translate('xpack.securitySolution.search.overview', {
+            defaultMessage: 'Overview',
+          }),
+        ],
+      },
+      {
+        id: SecurityPageName.detectionAndResponse,
+        title: DETECTION_RESPONSE,
+        path: DETECTION_RESPONSE_PATH,
+        features: [FEATURE.general],
+        keywords: [
+          i18n.translate('xpack.securitySolution.search.detectionAndResponse', {
+            defaultMessage: 'Detection & Response',
+          }),
+        ],
+      },
```

Comment:
`overview` and `detectionAndResponse` moved inside "Dashboards" so the global search results shows the hierarchy